root/static/scripts/edit/URLCleanup.js

```diff
@@ -501,7 +548,11 @@ const CLEANUPS = {
       new RegExp("^(https?://)?(www\\.)?avexnet\\.jp/id/[a-z0-9]{5}/discography/product/[A-Z0-9]{4}-[0-9]{5}\\.html$", "i"),
       new RegExp("^(https?://)?(www\\.)?kingrecords\\.co\\.jp/cs/g/g[A-Z]{4}-[0-9]+/$", "i")
     ],
-    type: LINK_TYPES.discographyentry
+    type: LINK_TYPES.discographyentry,
+    validate: function (url, id) {
+      // avoid wikipedia being added as release-level discography entry
+      return !/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url);
```

Comment:
I didn’t read your above message before posting mine, and I still don’t get exactly the idea.

The problem here is closely related to PR #362 that fixed PR #354.